clients/mtaclient/retryable_mta_rest_client.go

```diff
@@ -17,12 +17,12 @@ type RetryableMtaRestClient struct {
 
 func NewRetryableMtaRestClient(host, spaceID string, rt http.RoundTripper, jar http.CookieJar, tokenFactory baseclient.TokenFactory) RetryableMtaRestClient {
 	mtaClient := NewMtaClient(host, spaceID, rt, jar, tokenFactory)
-	return RetryableMtaRestClient{mtaClient: mtaClient, MaxRetriesCount: 3, RetryInterval: time.Second * 3}
+	return RetryableMtaRestClient{mtaClient: mtaClient, MaxRetriesCount: 18, RetryInterval: time.Second * 10}
```

Comment:
Why retries times were increased?

Reply:
This was just for testing, good catch, removing this